### PR TITLE
fix: Added type defs to PixelSpaceCallbackMapper

### PR DESCRIPTION
**Pull request by megumi-k**
Fix type definition for typescript.

### Context

[Error occurs when using vtkPixelSpaceCallbackMapper with TypeScript](https://discourse.vtk.org/t/error-occurs-when-using-vtkpixelspacecallbackmapper-with-typescript/12853)

### Results

Callback parameters work with TypeScirpt. 

### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: master
  - **OS**: Windows
  - **Browser**: Chrome

### CODE REVIEW

File: Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export interface vtkPixelSpaceCallbackMapper extends vtkMapper {
 	 * 
 	 * @param callback 
 	 */
-	setCallback(callback: () => any): boolean
+	setCallback(callback: (coords: any, camera: vtkCamera, aspect: number, depthValues: number[]) => any): boolean
```
Comment (daker) [inline Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts:70]: can you also update the documentation ?

Comment (megumi-k) [reply Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts]: I updated the documentation as below.

File: Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts [hunk 2]
```diff
@@ -65,9 +65,30 @@ export interface vtkPixelSpaceCallbackMapper extends vtkMapper {
...
+	 * 
+	 * const textActor = vtkActor.newInstance();
+	 * textActor.setMapper(psMapper);
+	 * renderer.addActor(textActor);
+	 * ```
+	 * 
+	 * @param callback called with coords, camera, aspect and depthBuffer
 	 */
-	setCallback(callback: () => any): boolean
+	setCallback(callback: (coords: any, camera: vtkCamera, aspect: number, depthBuffer: number[]) => any): boolean
```
Comment (floryst) [inline Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts:91]: `coords: any` should be `Vector3[]`.

Comment (megumi-k) [reply Sources/Rendering/Core/PixelSpaceCallbackMapper/index.d.ts]: Thank you for pointing it out.
I fixed wrong type defs.

**Pull request merged**